extensions/amp-story/1.0/media-tasks.js

```diff
@@ -355,9 +375,10 @@ export class UpdateSourcesTask extends MediaTask {
   /**
    * @param {!Sources} newSources The sources to which the media element should
    *     be updated.
+   * @param {!Object=} options
```

Comment:
Make this an explicit struct.

Reply:
I've been able to get rid of it, so we no longer have to make an exception in presubmit to load `usePromise`.